server/static/basic.html

```diff
@@ -92,7 +92,7 @@
         <!-- /Templates -->
       </a-assets>
 
-      <a-entity id="player" networked="template:#avatar-template;showLocalTemplate:false;" camera="userHeight: 1.6" spawn-in-circle="radius:3" wasd-controls look-controls>
+      <a-entity id="player" networked="template:#avatar-template;showLocalTemplate:false;" camera position="0 1.6 0" spawn-in-circle="radius:3" wasd-controls look-controls></a-entity>
```

Comment:
aframe vNext will not have a userHeight property on the camera. Instead, you need to set the position of the camera entity. 
Note that wasd-controls are also broken in VR mode, since WebVRManager has full control. This needs to be fixed by containing the camera in a camera/player rig. See https://github.com/aframevr/aframe/issues/3351